src/test/java/com/sonyericsson/jenkins/plugins/bfa/TransientCauseManagementHudsonTest.java

```diff
@@ -55,50 +58,49 @@
  *
  * @author Robert Sandell &lt;robert.sandell@sonyericsson.com&gt;
  */
-public class TransientCauseManagementHudsonTest extends HudsonTestCase {
+@WithJenkins
+class TransientCauseManagementHudsonTest {
```

Comment:
This class should technically change name now that it is no longer a `HudsonTestCase` but I don't care :)